### PR TITLE
Update `metaList.conditionals`

### PR DESCRIPTION
**Pull request by lantua**
## Describe your changes

The current `meta.ts` generates an incorrect `tag` value, leading to complication when trying to read the corresponding conditional values. This PR updates the `tag` so that the calculation simplifies to `calc.compute(member0.withTag(tag))`.

## Issue or discord link

- https://discord.com/channels/785153694478893126/1183809278209970217/1201416647974600875

## Testing/validation

n/a

## Checklist before requesting a review (leave this PR as draft if any part of this list is not done.)

- [X] I have commented my code, in hard-to understand areas.
- [X] I have made corresponding changes to README or wiki.
- [X] For front-end changes, I have updated the corresponding English translations.
- [X] Ran `yarn run mini-ci` locally to validate format + lint.
- [X] If I have added a new library or app, I have updated the deployment scripts to ignore changes as needed

### CODE REVIEW

File: libs/gi-formula/src/executors/gen-desc/executor.ts [hunk 1]
```diff
@@ -18,14 +18,16 @@ export default async function runExecutor(
...
-  for (const [src, entries] of Object.entries(conditionals!)) {
-    for (const [name, obj] of Object.entries(entries!)) {
-      entries![name] = { src, name, tag: { src, name }, ...obj }
-    }
+  const conditionals: Record<string, Record<string, any>> = {}
+  for (const { tag, meta } of metaList.conditionals!) {
+    const { src, q } = tag // Conditionals guarantee `src-q` pair uniqueness
+    if (!conditionals[src!]) conditionals[src!] = {}
+    if (!conditionals[src!][q!])
+      conditionals[src!][q!] = { src, name: q, tag, ...meta }
```
Comment (nguyentvan7) [inline libs/gi-formula/src/executors/gen-desc/executor.ts:29]:
```suggestion
      conditionals[src!][q!] = { src, q, tag, ...meta }
```
This allows easier usage of the tag later when fetching from the calc

Comment (nguyentvan7) [inline libs/gi-formula/src/executors/gen-desc/executor.ts:29]: should we add `qt: 'cond'`?

Comment (lantua) [reply libs/gi-formula/src/executors/gen-desc/executor.ts:29]: It's already in `tag`, (see the generated `meta.ts`). `src` and `name` are for UI use only.

Comment (nguyentvan7) [reply libs/gi-formula/src/executors/gen-desc/executor.ts:29]: Oh i see, tag value was also changed, nvm

Comment (nguyentvan7) [reply libs/gi-formula/src/executors/gen-desc/executor.ts:29]: nvm

**Review by nguyentvan7 — APPROVED** [review]

**Pull request merged**